src/org/gk/slicing/SlicingEngine.java

```diff
@@ -1406,7 +1693,7 @@ private static String getInput(String message) throws Exception {
         return input;
     }
     
-    @Test
+    //@Test
```

Comment:
Why all other test methods are disabled?

Reply:
My mistake, I shouldn't have disabled in the first place. Fixed in [5d0061c](https://github.com/reactome/CuratorTool/pull/20/commits/5d0061cf2e6db424efa382db7c7d1255250b379b).